index.js

```diff
@@ -39,6 +41,22 @@ let defaults = require('./defaults.js');
 // can be set to a number, which will effectively set the cluster
 // option to `cluster: { processes: n }`.
 //
+// `openTelemetryProvider`
+//
+// If set, Apostrophe will register it as a global OpenTelemetry tracer provider.
+// The expected value is an object, istance of TracerProvider.
```

Comment:
istance -> an instance

Reply:
An advanced setup would not involve OTel SDK (manual setup, manual initialization of resource, provider, exporters, etc). Adding an option to pass a provider gives the app implementation the freedom to do whatever is needed on an application level. The global provider registration is the only bridge between the core instrumentation and the application implementation.

Here is how it works (see the comments) - https://github.com/open-telemetry/opentelemetry-js-api/blob/main/src/trace/ProxyTracerProvider.ts#L33